src/VisualStudio/Core/Def/TaskList/VisualStudioTaskListService.cs

```diff
@@ -44,14 +44,15 @@ internal class VisualStudioTaskListService :
             IThreadingContext threadingContext,
             VisualStudioWorkspaceImpl workspace,
             IGlobalOptionService globalOptions,
-            IAsynchronousOperationListenerProvider asynchronousOperationListenerProvider,
+            ITableManagerProvider tableManagerProvider,
             SVsServiceProvider asyncServiceProvider,
+            IAsynchronousOperationListenerProvider asynchronousOperationListenerProvider,
             [ImportMany] IEnumerable<Lazy<IEventListener, EventListenerMetadata>> eventListeners)
         {
             _threadingContext = threadingContext;
             _workspace = workspace;
+            _tableManagerProvider = tableManagerProvider;
             _asyncServiceProvider = (IAsyncServiceProvider)asyncServiceProvider;
-            _eventListenerTracker = new EventListenerTracker<ITaskListProvider>(eventListeners, WellKnownEventListeners.TaskListProvider);
 
             _listener = new TaskListListener(
```

Comment:
Can the creation of this also be moved later?

Reply:
yes.  definitely could.  let me clean up more.